platform/integrations/rancher/rancher-integration.mdx

```diff
@@ -12,58 +12,102 @@ import Label from '@site/src/components/Label'
 
 import PartialRancher from "../../api/_partials/resources/projects/spec/rancher.mdx";
 
-Once Rancher is configured on vCluster Platform, virtual clusters can be imported into Rancher and started using as a cluster in Rancher. In order to enable this integration on each virtual cluster, the vCluster Platform project must be enabled with the Rancher integration first. 
+The vCluster Rancher Operator automatically imports vCluster installations as Rancher clusters. Rancher users can deploy virtual clusters without having cluster create permissions, as the operator handles the necessary setup. Additionally, project owners of the project vCluster is installed in and the cluster owners of the host cluster are automatically assigned as cluster owners in the new Rancher cluster.
 
-## Enable Rancher Integration on a vCluster Platform project
+After configuring Rancher within the vCluster Platform, you can import virtual clusters into Rancher and manage them as standard clusters.
```

Comment:
Remove this line please. The new operator currently has no dependence or integration with the platform apart from what you could do with any vcluster. You can import virtual clusters into rancher without the operator.